### ISSUE TITLE
@parcel/core: fablejs\.fable\fable-library.3.1.0\lib\long.js does not export 'Long'

### ISSUE THREAD

**Issue by albertwoo** [fable-compiler/Fable#2343]
### Description

I am using parcel 2.0.0-nightly.520, fable 3.1.0. I got this error for production build.
I am not familiar with js but if I move the line `export default Long;` after the `export function Long(` function then the errors goes away in file `fable-library.3.1.0\lib\long.js`.

**Comment by alfonsogarciacaro** [comment]
Ah, damn. I made a change in that file and maybe the exports are not "standard" now. Seems Webpack is sometimes more permissive so that may be the reason only Parcel is failing. Actually, I just realized we don't need the `export default Long` in that file. Does the error also go away if you just remove `export default Long`?

**Comment by ncave** [comment]
@alfonsogarciacaro I think a few files are still importing the default (e.g. BitConverter etc.), so they need to be fixed too.

**Comment by albertwoo** [comment]
@alfonsogarciacaro yes. Remove that line will also work.

**Comment by alfonsogarciacaro** [comment]
Thanks for confirming @albertwoo! I'll push a fix :+1:

Sorry @ncave, I should have given more context. Both Long and Decimal are "dual" modules: there's the public face in Typescript (Long.ts, Decimal.ts) and the private JS files (lib/long.js, lib/big.js) that we "borrowed" from external libraries. Other modules only access the public face and if I'm not mistaken the problem here was caused by lib/long.js so it should be ok to remove the default export because it's not used by Long.ts.

To give more context, I changed the file in https://github.com/fable-compiler/Fable/commit/8441abe566224d957ccd9d50988dd92a084e4b37 to prevent importing Long and Decimal when using string formatting. Some additional info can also be found here: https://github.com/thoth-org/Thoth.Json/pull/96
